composer.json

```diff
@@ -39,7 +40,8 @@
     "omines/oauth2-gitlab": "3.5.0",
     "pear/archive_tar": "1.4.14",
     "php-di/php-di": "7.0.5",
-    "ramsey/uuid": "4.7.4"
+    "ramsey/uuid": "4.7.4",
+    "socialiteproviders/github": "^4.1"
```

Comment:
We use Dependabot to keep `composer.json` updated, so it's better to pin every dependency to the most specific version possible.  Also apply above...